src/qml/LoginPage.qml

```diff
@@ -1,50 +1,65 @@
-import QtQuick 2.5
-import QtQuick.Controls 1.2
+import QtQuick 2.0
+import QtQuick.Controls 1.2 as Controls
+import QtQuick.Layouts 1.2
+import org.kde.kirigami 1.0 as Kirigami
 import harbour.kaidan 1.0
 
-Item {
-	Column {
-		Row {
-			TextField {
-				id: jidField
-				width: 150
-				placeholderText: "JID"
-			}
-			TextField {
-				id: passField
-				width: 150
-				height: 27
-				placeholderText: "Password"
-				echoMode: TextInput.Password
-			}
+Kirigami.ScrollablePage {
+	id: page
+	title: "Login"
+	Layout.fillWidth: true
+	implicitWidth: applicationWindow().width
+
+	ColumnLayout {
+		width: parent.width
+
+		Controls.Label {
```

Comment:
Login page looks a bit ugly. Please, put Label and TextField into a container like RowLayout.
Like this:
```
		RowLayout {
			Controls.Label {
				text: "Your Password:"
			}
			Controls.TextField {
				id: passField
				placeholderText: "Password"
				echoMode: TextInput.Password
				Layout.minimumWidth: parent.width * 1.5
				Layout.alignment: Qt.AlignHCenter
			}
		}
```